backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/util/FreinetCreatePersonApi.kt

```diff
@@ -0,0 +1,124 @@
+package app.ehrenamtskarte.backend.freinet.util
+import app.ehrenamtskarte.backend.exception.webservice.exceptions.RegionNotFoundException
+import app.ehrenamtskarte.backend.regions.database.repos.RegionsRepository
+import com.fasterxml.jackson.databind.JsonNode
+import io.ktor.client.HttpClient
+import io.ktor.client.plugins.HttpRequestRetry
+import io.ktor.client.plugins.HttpResponseValidator
+import io.ktor.client.request.request
+import io.ktor.client.request.setBody
+import io.ktor.client.statement.bodyAsText
+import io.ktor.http.ContentType
+import io.ktor.http.HttpMethod
+import io.ktor.http.HttpStatusCode
+import io.ktor.http.URLProtocol
+import io.ktor.http.contentType
+import io.ktor.http.path
+import kotlinx.coroutines.runBlocking
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class FreinetCreatePersonApi(private val host: String) {
```

Comment:
Perhaps it would be enough to have FreinetApi class or FreinetClient and put all functions there?